src/main/java/org/worshipsongs/importer/SongParser.java

```diff
@@ -53,15 +82,125 @@ public String parseAttribute(String input, String attributeName)
+            DocumentBuilder docBuilder = docFactory.newDocumentBuilder();
+            Document document = docBuilder.newDocument();
+            document.setXmlStandalone(true);
+
+            Element songElement = getSongElement(document);
+            document.appendChild(songElement);
+
+            Element lyricsElement = getLyricsElement(document);
+            songElement.appendChild(lyricsElement);
+
+            String verseOrders[] = splitVerseOrder(verseOrder);
+            String verses[] = splitVerse(lyrics);
+
+            for (int i = 0; i < verseOrders.length; i++)
+            {
+                Element elementVerse = getVerseElement(document, verseOrders[i], verses[i+1]);
+                lyricsElement.appendChild(elementVerse);
+            }
+            Transformer transformer = TransformerFactory.newInstance().newTransformer();
+            transformer.transform(new DOMSource(document), new StreamResult(out));
+
+        } catch (ParserConfigurationException e) {
+            logger.log(Level.SEVERE, "Exception occurs in:", e);
+        } catch (TransformerException e) {
+            logger.log(Level.SEVERE, "Exception occurs in:", e);
+        }
+        return out.toString();
+    }
+
+    Element getSongElement(Document document)
+    {
+        Element song = document.createElement("song");
+        Attr version = document.createAttribute("version");
+        version.setValue("1.0");
+        song.setAttributeNode(version);
+        return song;
+    }
+
+    Element getLyricsElement(Document document)
+    {
+        return document.createElement("lyrics");
+    }
+
+    Element getVerseElement(Document document, String verseOrders, String verse)
+    {
+        Element verseElement = document.createElement("verse");
+        Attr type = document.createAttribute("type");
+        type.setValue(splitVerseType(verseOrders));
+        verseElement.setAttributeNodeNS(type);
+        Attr label = document.createAttribute("label");
+        label.setValue(splitVerseLabel(verseOrders));
+        verseElement.setAttributeNodeNS(label);
+        verseElement.appendChild(document.createCDATASection(verse.trim()));
+        return verseElement;
+    }
+
+    String[] splitVerseOrder(String verseOrder)
+    {
+        return verseOrder.split(" ");
+    }
+
+    String splitVerseType(String verse)
+    {
+        String verseType = "";
+        if(!verse.isEmpty()) {
+            verseType = verse.split("")[1].toLowerCase();
+        }
+        return verseType;
+    }
+
+    String splitVerseLabel(String verse)
+    {
+        String verseLabel = "";
+        if(!verse.isEmpty()) {
+            verseLabel = verse.split("")[2];
+        }
+        return verseLabel;
+    }
+
+    String[] splitVerse(String lyrics)
+    {
+        return lyrics.split("\\[..\\]");
+    }
+
+    String parseSongBook(String input)
+    {
+        return parseAttribute(input, "songBook");
+    }
+
+    Song parseSong(String input)
```

Comment:
This should be a "public" method. Define this as the first method in this class.